### ISSUE TITLE
Install error: Unable to load valid system manifest

### ISSUE THREAD

**Issue by KatiaSae** [OtherwiseJunk/ParanoiaPerfectEditionSystem-foundryvtt#28]
When trying to install via "Game Systems" the following error appears:

Error: Unable to load valid system manifest data from "https://github.com/OtherwiseJunk/ParanoiaPerfectEditionSystem-foundryvtt/blob/main/system.json" System validation errors:id:may not be undefined

**Comment by OtherwiseJunk** [comment]
Hey @KatiaSae thanks for the bug report.

I believe I've found the issue and resolved it, but it may take some time for the cache to update on the foundry site. 

https://github.com/OtherwiseJunk/ParanoiaPerfectEditionSystem-foundryvtt/releases/download/v0.1.0/system.json

Short-term, this link can be used to manually install the module, I've confirmed this works from my own foundryvtt system.

This evening I'll be testing installing from the foundry store and I won't close this issue out until I can confirm I can pull the game from there.
